yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -1521,6 +1583,95 @@ module vyatta-protocols-frr-bgp-v1 {
 							}
 						}
                                         }
+
+					container ipv4-vpn {
+						description "BGP neighbor parameters";
+						presence "Enables BGP neighbor IPv4 parameters";
+						configd:help "BGP neighbor parameters";
+						uses peer-grp-neighbor-afi-common-settings {
+							refine filter-list/export {
+								must "not(../../peer-group)" {
+									error-message "You may not configure filter-list export for a neighbor in peer-group\n";
+								}
+							}
+							refine route-map/export {
+								must "not(../../peer-group)" {
+									error-message "You may not configure route-map export for a neighbor in peer-group\n";
+								}
+							}
+							refine attribute-unchanged {
+								must "not(../peer-group)" {
+									error-message "You may not configure attribute-unchanged for a neighbor in peer-group\n";
+								}
+							}
+							refine remove-private-as {
+								must "not(../peer-group)" {
+									error-message "You may not configure remove-private-as for a neighbor in peer-group\n";
+								}
+							}
+							refine route-reflector-client {
+								must "not(../peer-group)" {
+									error-message "You may not configure route-reflector-client for a neighbor in peer-group\n";
+								}
+							}
+							refine disable-send-community {
+								must "not(../peer-group)" {
+									error-message "You may not configure disable-send-community for a neighbor in peer-group\n";
+								}
+							}
+							refine capability/orf {
+								must "not(../../peer-group)" {
+									error-message "You may not configure capability orf for a neighbor in peer-group";
+								}
+							}
+							refine default-originate {
+								must "not(../peer-group)" {
+									error-message "You may not configure default-originate for a neighbor in peer-group\n";
+								}
+							}
+							refine unsuppress-map {
+								must "not(../peer-group)" {
+									error-message "You may not configure unsuppress-map for a neighbor in peer-group\n";
+								}
+							}
+							refine nexthop-self {
+								must "not(../peer-group)" {
+									error-message "You may not configure nexthop-self for a neighbor in peer-group\n";
+								}
+							}
+							refine route-server-client {
+								must "not(../peer-group)" {
+									error-message "You may not configure route-server-client for a neighbor in peer-group\n";
+								}
+							}
+						}
+						uses neigh_peer_grp_ipv4_lists {
```

Comment:
For future reference, please always use hyphens in names, never underscores.

Reply:
Okay we will follow that. 
But here the grouping is already mentioned in this way so we just used in our container ipv4-vpn.

Thanks.

Reply:
Wasn't asking you to change *existing* code - that's unnecessary churn.  Just making a recommendation for naming in future as a matter of style / consistency.

Reply:
Okay got it. 

Thanks,